app/include/zmk/split/bluetooth/central.h

```diff
@@ -5,4 +5,7 @@
 #include <zmk/behavior.h>
 
 int zmk_split_bt_invoke_behavior(uint8_t source, struct zmk_behavior_binding *binding,
-                                 struct zmk_behavior_binding_event event, bool state);
\ No newline at end of file
+                                 struct zmk_behavior_binding_event event, bool state);
+
+bool zmk_split_bt_central_peripheral_is_connected(uint8_t index);
+bool zmk_split_bt_central_peripheral_is_bonded(uint8_t index);
```

Comment:
Thoughts on a single API that returns an enum with the state? The "is connected" makes no sense if we don't even have an address bonded to that index, for example, so a single API to just query the "peripheral state" makes more sense, IMHO.

Reply:
I think that's a good idea in general, my only concern would be that's a divergence from how it's currently handled for central->host connections and peripheral->central connections which both use a pair of bool return functions. Would it be better to change all of them over to a consistent API with one `BLE_CONN_STATE` enum with connected, bonded but not connected and unbounded values?

Reply:
Thinking about this more i'm concerned changing the other functions would leak to breakage of module and config repo custom display implementations. A stopgap would be implement enum return, then change the bool return functions to reference that enum return function for some period of time then remove them, similar to how the encoder changes were handled